### ISSUE TITLE
Zoom?

### ISSUE THREAD

**Issue by matthiasbeyer** [deinstapel/cursive-multiplex#1]
Hi!

Does this crate feature something like tmux'es "zoom" feature? If not, please consider this a feature request!

**Comment by jwuensche** [comment]
Hi,
first of all sorry for responding late, I completely missed your issue in my mails.
no the crate does not support something like this at the moment, but it is great idea to add something like this. It should be possibly to implement this quite quickly.

I'm gonna add this as a feature request and see if I can add it over the holidays :D

**Comment by matthiasbeyer** [comment]
Would love to see this, yeah! Maybe I can contribute some code, too... Let's see what 36c3 brings ... :-)

**Comment by jwuensche** [comment]
Managed to implement the feature today, it is currently with tests on master. I will release a version once I tested a few more more scenarios and am happy.
You're welcome to try it out and correct anything I missed :sweat_smile:.

Changes API-wise there are just new setters to define the event the zoom should occur on - `with_zoom` and `set_zoom`.

Also adjusted the README that it includes the default key for zooming (atm `Ctrl` + `x`).

Commits: 36954e18f3cb2ad7eb8984adca4d18f00f5aa0ea 6b2948f4d1b18dbfbf965a549b6e372cf024a56d b85dad4e07b9382dc861d18827ad4b2f2c47a7af
